pkg/scalers/authentication_types.go

```diff
@@ -0,0 +1,9 @@
+package scalers
```

Comment:
minor nit, could you please move this file to the separate package, eg. `scalers/authentication` or something like that